app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
```

Comment:
Sorry, you're right. During the channel talk I only asked about `confirmable` and assumed all defaults should stay as they are. But I remember now we only agreed on those basic ones.